lib/card.rb

```diff
@@ -38,4 +38,25 @@ class Card < ActiveRecord::Base
   TRACKED_FIELDS = %w(name type_id db_content trash)
   Loader.load_mods if count > 0
 
+
+  def self.find_each(options = {})
+    if block_given?
+      find_in_batches(options) do |records|
+        records.each { |record| yield record }
+      end
+    else
+      super(options)
+    end
+  end
```

Comment:
Hmm, I wouldn't think so.  Being a class method in a module means it will be attached to the class with extend (when included), but when it runs the method, what is self?  It should be Card (the class).  I can't really see why this could be happening.